src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -97,7 +97,11 @@ describe('Test suite for dictionary concept actions', () => {
       const request = moxios.requests.mostRecent();
       request.respondWith({ status: 200, response: expectedPosts });
     });
-    await fetchSourceConcepts('source', 'query');
+    try {
+      await fetchSourceConcepts('source', 'query');
```

Comment:
What is this testing?

Reply:
After fixing this [PR](https://github.com/openmrs/openmrs-ocl-client/pull/392), the coverage dropped. So this test is meant to raise the coverage for the particular line that drooped in coverage.

Reply:
That does not seem to address my question

Reply:
In the "fetchSourceConcepts" action in the "dictionaryConcepts" file, there is a "catch" clause that dropped coverage. The test is meant to account for the occurrence of an error in the try-catch statement.

Reply:
Is that what is meant by line 103 below?

Reply:
Yes